src/dbos-executor.ts

```diff
@@ -837,22 +856,22 @@ export class DBOSExecutor implements DBOSExecutorContext {
           return DBOSJSON.stringify(recordedResult) === DBOSJSON.stringify(callResult);
         }
 
-        internalStatus.output = result;
+        internalStatus.output = DBOSJSON.stringify(result);
         internalStatus.status = StatusString.SUCCESS;
         if (internalStatus.queueName && !this.isDebugging) {
           // Now... the workflow isn't certainly done.
```

Comment:
This comment confuses me (I know it's old)&mdash;if I understand correctly, with our recent changes, at this point we know the workflow is done and can safely dequeue it and record its output.

Reply:
Transactionality we'll do later, and it's not directly relevant here (more for starting workflows). I think the comment is outdated and can be removed, not a huge deal.